.github/workflows/release_to_pypi.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish pyogrio to PyPI / GitHub
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  build-n-publish:
+    name: Build and publish pyogrio to PyPI
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Build a source tarball
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools
```

Comment:
@brendan-ward we probably need also Cython in here, don't we?

Reply:
In principle, cython is not needed, because that is specified in the pyproject.toml

Reply:
We should also all the `build` package here, instead of directly invoking setup.py